### DISCUSSION TITLE
Can't get correct still image URL to work

### DISCUSSION THREAD

**Discussion by rw377** [tsightler/ring-mqtt#474] [Support Q&A]
I am setting up the generic camera in HA and got the RTSP stream working. However, I would like to add the 'still image' during the generic cam configuration so the live stream isn't always trying to turn on.  
The suggested url is: `https://homeassistant:8123{{ states.camera.garage_snapshot.attributes.entity_picture }}` 
but i have changed it to:`https://myha.duckdns.org{{ states.camera.garage_snapshot.attributes.entity_picture }}` which is the URL I use to access my HA externally. 
I get an error of: 
> Problem rendering template Template("https://myha.duckdns.org{{ states.camera.garage_snapshot.attributes.entity_picture }}"): UndefinedError: 'None' has no attribute 'attributes'

Should I not be using this domain? I am a bit confused here and I've read up on all of the documentation I could. I', sure its an easy fix but any help would be appreciated!
Edit: I have also unchecked self signed SSL to no avail, and this shouldn't be necessary anyways as I used a Let's Encrypt cert

**Comment by tsightler** [comment]
> UndefinedError: 'None' has no attribute 'attributes'

This error implies that the connection to the API is being made but the attribute itself doesn't exist.  Do you see the snapshot camera in the UI?  The template above is basically just a way to get the Generic IP camera entity to automatically use the HA camera proxy for the still image.  If the snapshot camera isn't working, or you haven't yet retrieved a snapshot from this camera (for example, battery cameras only send a snapshot on motion by default), then this error could happen.

**Reply by rw377** [reply]
Thanks for the quick reply here. It looks like it was just a typo on my end in a very odd way. Thanks again.
